srcopsmetrics/bot_knowledge.py

```diff
@@ -53,6 +54,12 @@ def analyse_projects(projects: List[Tuple[str, str]], is_local: bool = False) ->
         project_path = path.joinpath("./" + github_repo.full_name)
         check_directory(project_path)
 
+        if entities is not None:
+            for entity in entities:
+                _LOGGER.info("%s inspection" % entity)
+                github_knowledge.analyse_entity(github_repo, project_path, entity, is_local)
+            continue
+
```

Comment:
```
         if entities is not None:
            for entity in entities:
                _LOGGER.info("%s inspection" % entity)
                github_knowledge.analyse_entity(github_repo, project_path, entity, is_local)
            continue

        _LOGGER.info("Issues inspection")
        github_knowledge.analyse_entity(github_repo, project_path, EntityTypeEnum.ISSUE.value, is_local)

        _LOGGER.info("Pull requests inspection")
        github_knowledge.analyse_entity(github_repo, project_path, EntityTypeEnum.PULL_REQUEST.value, is_local)

        _LOGGER.info("Content Files inspection")
        github_knowledge.analyse_entity(github_repo, project_path, EntityTypeEnum.CONTENT_FILE.value, is_local)
```

It feels redundant, what about reducing all that code. I'm going to send you a PR in this PR one moment.

Reply:
ack